flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/MathFunctionsITCase.java

```diff
@@ -139,6 +139,34 @@ Stream<TestSetSpec> getTestSetSpecs() {
                                 $("f0").truncate(2),
                                 "TRUNCATE(f0, 2)",
                                 new BigDecimal("123.45"),
-                                DataTypes.DECIMAL(6, 2).notNull()));
+                                DataTypes.DECIMAL(6, 2).notNull()),
+                TestSetSpec.forFunction(BuiltInFunctionDefinitions.SHIFTLEFT)
+                        .onFieldsWithData(Long.valueOf("123456"))
+                        .testResult(
+                                $("f0").shiftleft(2L),
+                                "shiftleft (f0, 2)",
+                                Long.valueOf("493824"),
+                                DataTypes.BIGINT().notNull()),
+                TestSetSpec.forFunction(BuiltInFunctionDefinitions.SHIFTLEFT)
+                        .onFieldsWithData(Long.valueOf("123456"))
+                        .testResult(
+                                $("f0").shiftleft(2L),
+                                "shiftleft (f0, 2)",
+                                Long.valueOf("493824"),
+                                DataTypes.BIGINT().notNull()),
```

Comment:
These two tests seem the same for the input.

Reply:
Fixed.